examples/helloworld/tf_example5/README.md

```diff
@@ -16,66 +16,33 @@ We also prepared related scripts in [TF image_recognition example](../../tensorf
 wget https://storage.googleapis.com/intel-optimized-tensorflow/models/v1_6/mobilenet_v1_1.0_224_frozen.pb
 ```
 
-### 4. Update the root of dataset in conf.yaml
```

Comment:
Please also update the description "This example is used to demonstrate how to config benchmark **in yaml** for performance measurement."